web/src/app/services/custom-fields/taskana-customization-test.json

```diff
@@ -0,0 +1,22 @@
+{
+  "EN": {
+      "classifications": {
+          "information": {
+              "custom1": {
+                  "field": "Classification custom 1",
+                  "visible": true
+              },
+              "custom3": {
+                  "field": "",
+                  "visible": false
+              }
+          },
+          "categories": {
+              "EXTERNAL": "assets/icons/categories/external.svg",
```

Comment:
Isn't there DEFAULT missing?

Reply:
No, it isn't necessary. In this file, we only put the custom icons. But if you prefer I can put the default icon